pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
As written it would rewrite this file:

```
# hello
world

# down
here
```

to this
```
# down
# hello
here
world
```

(instead of)

```
# down
here
# hello
world
```

which seems non-ideal (whereas `requirements-txt-sorter` would preserve the chunking and comments)